dvc/testing/benchmarks/fixtures.py

```diff
@@ -99,7 +98,8 @@ def parse_tuple(version_string):
     return (parsed.major, parsed.minor, parsed.micro)
 
 
-def _skip_if_dvc_version_lt_required(request, dvc_bin):
+@pytest.fixture
+def dvc_bin(request, make_dvc_bin):
```

Comment:
This used to be a separate `autouse=True` fixture, but we don't want it to be autouse (since it forced the session level `dvc_bin` venv to be created and parametrized every test based on `dvc_rev`, even if we aren't running any benchmark tests)